models/classes/media/LocalItemSource.php

```diff
@@ -179,6 +180,13 @@ public function delete($filename)
 
         return $deleted;
     }
+    
+    public function getFileStream($link)
+    {
+        $sysPath = $this->getSysPath($link);
+        $fh = fopen($sysPath, 'r');
+        return new Stream($fh, ['size' => filesize($sysPath)]);
```

Comment:
Good catch, parameters belonged to a different stream implementation, used earlier